lms/djangoapps/support/static/support/jsx/single_support_form.jsx

```diff
@@ -183,8 +193,67 @@ class RenderForm extends React.Component {
     );
   }
 
+  async handleInputChange(event) {
+    event.preventDefault();
+    const queryString = event.target.value;
+    if (queryString.length > 2) {
+      let suggestions = [];
+      await fetch(`${this.learnerSupportCenterURL}/hc/api/internal/instant_search.json?query=${queryString}&locale=en-us`, { headers: {
```

Comment:
why length > 2? Plus, you will be hitting api after every key input. I would say adding some time based checks to avoid hitting the API unless user stops typing.

Reply:
Ohh, right. I'll add a timeout on this.
I looked into https://support.edx.org/hc/en-us and it was sending API calls at whenever we have a querystring with length more than 3.